test/viewmodels/SecuritiesRequestedStatementsViewModelSpec.scala

```diff
@@ -0,0 +1,192 @@
+      val row = StatementRow(
+        0,
+        None,
+        "10 July 2023 to 20 July 2023",
+        Some(pdfLink),
+        "rowId",
+        "dateCellId",
+        "linkCellId"
+      )
+
+      val result = SecuritiesRequestedStatementsViewModel.renderEoriHeading(row)
+
+      result mustBe None
+    }
+  }
+
+  "renderPdfLink" should {
+    "render the correct PDF link" in {
+      val pdf = Some(pdfLink)
+      val result = SecuritiesRequestedStatementsViewModel.renderPdfLink(pdf)
+
+      val expectedHtml = spanLinkComponent(
+        msg = s"PDF (${pdfLink.formattedSize})",
+        url = pdfLink.downloadURL,
+        classes = "file-link govuk-link",
+        spanClass = Some("govuk-visually-hidden"),
+        spanMsg = Some("Download PDF")
+      ).body.trim
+
+      result.body.trim.replaceAll("\\s+", " ") mustBe expectedHtml.replaceAll("\\s+", " ")
+    }
+  }
+
+  "renderMissingDocumentsGuidance" should {
+    "generate guidance" in {
+      val result: HtmlFormat.Appendable = SecuritiesRequestedStatementsViewModel.renderMissingDocumentsGuidance
+
+      val document: Document = Jsoup.parse(result.body)
+
+      val heading: Elements = document.select("#missing-documents-guidance-heading")
+
+      heading.size() mustBe 1
+    }
+  }
+
+  "hasSecurityStatementsForEori" should {
+    "return true if there are requested statements" in {
+      val securityStatements = Seq(
+        SecurityStatementsForEori(
+          eoriHistory = EoriHistory("EORI456", Some(LocalDate.parse("2023-07-10")), Some(LocalDate.parse("2023-07-20"))),
+          currentStatements = Seq(requestedStatement),
+          requestedStatements = Seq(requestedStatement)
+        )
+      )
+
+      val result = SecuritiesRequestedStatementsViewModel.hasSecurityStatementsForEori(securityStatements)
+
+      result mustBe true
+    }
+
+    "return false if there are no requested statements" in {
+      val securityStatements = Seq(
+        SecurityStatementsForEori(
+          eoriHistory = EoriHistory("EORI456", Some(LocalDate.parse("2023-07-10")), Some(LocalDate.parse("2023-07-20"))),
+          currentStatements = Seq(requestedStatement),
+          requestedStatements = Seq.empty
+        )
+      )
+
+      val result = SecuritiesRequestedStatementsViewModel.hasSecurityStatementsForEori(securityStatements)
+
+      result mustBe false
+    }
+  }
+}
+
+trait Setup extends SpecBase {
+  implicit val mockMessages: Messages = mock[Messages]
+
+  val pdfLink: PdfLink = PdfLink("file.pdf", "1MB", "Download PDF")
+
+  val requestedStatement: SecurityStatementsByPeriod = SecurityStatementsByPeriod(
+    startDate = LocalDate.parse("2023-07-10"),
+    endDate = LocalDate.parse("2023-07-20"),
+    files = Seq.empty
+  )
+
+  val securityStatementForEori: SecurityStatementsForEori = SecurityStatementsForEori(
+    eoriHistory = EoriHistory("EORI456", Some(LocalDate.parse("2023-07-10")), Some(LocalDate.parse("2023-07-20"))),
```

Comment:
Better to create vals for "2023-07-10" and "2023-07-20" as its been used at lots of places. Also better to create a val for " "